guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
+= Changing the Repository Sets Status in {Project}
+
+Repository sets show available repositories to each host.
+If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```

Comment:
No need to repeat this here (it's more relevant below.) But instead it might be good to expand a bit on the overview of what Repository sets is used for.

```suggestion
Repository sets show repositories available to each host. A host will be able to access content from a given repository if that repository is enabled.
```